terraform/.terraform/plugins/windows_amd64/lock.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
https://github.com/github/gitignore/blob/master/Terraform.gitignore 
add this to the gitignore an remove this file from git

git rm -r --cached {foldername}